docs/gallery_tags.rst

```diff
@@ -0,0 +1,266 @@
+- `Binomial <./distributions/gallery/binomial.html>`_
+- `Categorical <./distributions/gallery/categorical.html>`_
+- `Discrete Uniform <./distributions/gallery/discrete_uniform.html>`_
+- `Discrete Weibull <./distributions/gallery/discrete_weibull.html>`_
+- `Geometric <./distributions/gallery/geometric.html>`_
+- `Hypergeometric <./distributions/gallery/hypergeometric.html>`_
+- `Negative Binomial <./distributions/gallery/negativebinomial.html>`_
+- `Poisson <./distributions/gallery/poisson.html>`_
+- `Zero-Inflated Binomial <./distributions/gallery/zeroinflatedbinomial.html>`_
+- `Zero-Inflated Negative Binomial <./distributions/gallery/zeroinflatednegativebinomial.html>`_
+- `Zero-Inflated Poisson <./distributions/gallery/zeroinflatedpoisson.html>`_
+
+For more on discrete probability distributions, see:  
+`Probability mass function  <https://en.wikipedia.org/wiki/Probability_mass_function>`_.
+
+For more on univariate probability distributions, see:  
+`Probability distribution  <https://en.wikipedia.org/wiki/Probability_distribution>`_.
+
+.. _modifiers:
+
+Modifiers (Special Cases)
+-------------------------
+
+These are special modifications or cases applied to a “base” distribution:
+
+- `Censored <./distributions/gallery/censored.html>`_
+- `Hurdle <./distributions/gallery/hurdle.html>`_
+- `Mixture <./distributions/gallery/mixture.html>`_
+- `Truncated <./distributions/gallery/truncated.html>`_
+
+For more details on distribution modifications, see:  
+`Censoring (statistics) <https://en.wikipedia.org/wiki/Censoring_(statistics)>`_,  
+`Hurdle model  <https://en.wikipedia.org/wiki/Hurdle_model>`_,  
+`Mixture model  <https://en.wikipedia.org/wiki/Mixture_model>`_, and  
+`Truncated distribution  <https://en.wikipedia.org/wiki/Truncated_distribution>`_.
+
+.. _heavy_tailed:
+
+Heavy-Tailed
+------------
+
+Distributions with tails that decay slowly (i.e. they allow for large outliers):
+
+- `Cauchy <./distributions/gallery/cauchy.html>`_
+- `Student's t <./distributions/gallery/students_t.html>`_
+- `Pareto <./distributions/gallery/pareto.html>`_
+- `Half-Cauchy <./distributions/gallery/halfcauchy.html>`_
+- `Half-Student's t <./distributions/gallery/halfstudentt.html>`_
+- `Inverse Gamma <./distributions/gallery/inversegamma.html>`_
+
+For more on heavy-tailed distributions, see:  
+`Heavy-tailed distribution  <https://en.wikipedia.org/wiki/Heavy-tailed_distribution>`_.
+
+.. _light_tailed:
+
+Light-Tailed
+------------
+
+Distributions with tails that decay relatively quickly:
+
+- `Normal <./distributions/gallery/normal.html>`_
+- `Exponential <./distributions/gallery/exponential.html>`_
+- `Gamma <./distributions/gallery/gamma.html>`_
+- `Weibull <./distributions/gallery/weibull.html>`_
+- `Logistic <./distributions/gallery/logistic.html>`_
+
+For more on tail behavior and light-tailed distributions, see (by contrast with heavy tails):  
+`Heavy-tailed distribution  <https://en.wikipedia.org/wiki/Heavy-tailed_distribution>`_.
```

Comment:
I think we can not link to, but we can mention that distributions have a `kurtosis` method.